xml/smtp_banners.xml

```diff
@@ -1713,13 +1820,21 @@ The system or service fingerprint with the highest certainty overwrites the othe
       <param pos="4" name="system.time"/>
     </fingerprint>
 
-   <fingerprint pattern="^([^ ]+) E?SMTP(?: Ready\.?)?$">
-     <description>
-       catch all for daemons that have no distinguishing fingerprint whatsoever
-     </description>
-      <example>foo.example.com ESMTP</example>
-      <example>foo.example.com ESMTP Ready</example>
-      <example>foo.example.com SMTP</example>
+    <fingerprint pattern="^(\S+) E?SMTP Perl" flags="REG_ICASE">
```

Comment:
What I think you are elluding to is what I meant to do -- asserting `host.name` is useful but just Perl for the service fingerprint isn't useful (at least not today).  I could go either way.